tests/components/searchDrug/View.test.jsx

```diff
@@ -39,20 +42,31 @@ describe('Component: searchDrug: View', () => {
 
     it('show search error...', () => {
       props.error = { data: { message: 'Network error' } };
-      const component = getComponent().find('span').at(2);
+      const component = getComponent().find('span').at(1);
 
       expect(component.props().children).toBe('Network error');
     });
   });
 
-  describe('Search drug action', () => {
-    it('calls setCurrentLocation()', () => {
+  describe('Drug Search input field', () => {
+    it('calls onChange()', () => {
       const event = { target: { value: 'para' } };
-      const searchDrugSpy = jest.spyOn(props, 'searchDrug');
-      const drugSearchInput = getComponent().find('p input').first();
+      const searchDrugSpy = jest.spyOn(props, 'onChange');
+      const drugSearchInput = getComponent().find('span input').first();
       drugSearchInput.simulate('change', event);
 
       expect(searchDrugSpy).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Select drug action', () => {
+    it('calls onOptionSelected()', () => {
+      props.focused = true;
+      const optionSelectedSpy = jest.spyOn(props, 'onOptionSelected');
+      const drugSearchInput = getComponent().find('.option').first();
+      drugSearchInput.simulate('click');
+
```

Comment:
do we need this space inside an it block ?

Reply:
I was considering using white spaces for code readability, but I think it's not necessary

Reply:
Thanks